src/SystemAbstractions/Abstractions/ISystem.cs

```diff
@@ -2,9 +2,9 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 using System;
 
-namespace Axe.Windows.Abstractions
+namespace SystemAbstractions
```

Comment:
>SystemAbstractions [](start = 10, length = 18)

I'd argue that this namespace belongs under Axe.Windows, so Axe.Windows.SystemAbstractions. That's also the name of the assembly, so it's consistent with the pattern that we follow elsewhere.